app/src/main/java/com/android/unio/MainActivity.kt

```diff
@@ -51,7 +51,8 @@ fun UnioApp() {
   val db = FirebaseFirestore.getInstance()
 
   val associationRepository = AssociationRepositoryFirestore(Firebase.firestore)
-  val associationViewModel = AssociationViewModel(associationRepository)
+  val eventRepository = EventRepositoryFirestore(Firebase.firestore)
```

Comment:
This should be refactored to use the db defined above, but I've fixed this in my PR so it should be fixed by merging